lighthouse-core/audits/largest-contentful-paint-node.js

```diff
@@ -0,0 +1,80 @@
+/**
+ * @license Copyright 2020 The Lighthouse Authors. All Rights Reserved.
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License. You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License.
+ */
+'use strict';
+
+const Audit = require('./audit.js');
+const i18n = require('../lib/i18n/i18n.js');
+
+const UIStrings = {
+  /** Descriptive title of a diagnostic audit that provides the element that was determined to be the Largest Contentful Paint. */
+  title: 'Largest Contentful Paint element',
```

Comment:
maybe the id/filename should also be `*-element`, instead of node?

Reply:
On this note, should I just go ahead and rename `trace-nodes` to be `trace-elements` as well?

Reply:
I think so. in HTML, everything is a node, even attributes, comments, text nodes, etc ... so even `devtoolsNodePath` should have been `devtoolsElementPath` (or support a way to further specify an attribute or text node ... paul says it does support text nodes tho)

since these trace nodes are only gonna reference element nodes, yeah I like renaming to `trace-elements`.

Reply:
@paulirish

Reply:
yeah calling this traceElements works for me. lets change to that.


lol even `getNodeSelector` and `getNodeLabel` are documented as taking Element/HTMLElement and will fail if passed a textnode/comment node.